packages/FreiFahren_BE-NLP/db_utils.py

```diff
@@ -53,17 +53,20 @@ def insert_ticket_info(
 
     # Prepare the JSON data payload
     data = {
+        'timestamp': timestamp.isoformat(),
         'line': line,
         'station': station_name,
         'direction': direction_name,
         'author': None,
-        'message': message
+        'message': 'from bot'
```

Comment:
Can you Ignore this message in getRecentTicketInspectors so that we can use this an identifier for messages that are coming from the bot and messages coming from the user but still avoid displaying it to the user.